lib/europeana/mixins/image_version.rb

```diff
@@ -37,7 +37,8 @@ def versions(name = 'image')
           if image&.essence&.picture.present?
             VERSIONS.each_with_object({}) do |(version, settings), memo|
```

Comment:
In the definition of `VERSIONS`, why are the `quality` keys sometimes just `quality`, somtimes `'quality'`?

Reply:
no idea, I'll update them to be consistent though.